### ISSUE TITLE
Dynamic filters cause fatal error on pages with 2 different struct aggregations

### ISSUE THREAD

**Issue by micgro42** [cosmocode/dokuwiki-plugin-struct#107]
The bug happens on a pages which have two struct aggregations both with dynamic filters enabled and showing the output of different schemas.

If on one table a dynamic filter is apllied then the page reloads with the filter argument in the url. However that dynamic filter is then applied to all tables on the page, causing a fatal error in the other table.

The relevant part of the stacktrace:
```
16	0.0738	9108336	syntax_plugin_struct_table->render( )	.../renderer.php:105
17	0.0738	9108592	dokuwiki\plugin\struct\meta\SearchConfig->__construct( )	.../table.php:99
18	0.0743	9149248	dokuwiki\plugin\struct\meta\SearchConfigParameters->__construct( )	.../SearchConfig.php:57
19	0.0743	9149360	dokuwiki\plugin\struct\meta\SearchConfigParameters->addFilter( )	.../SearchConfigParameters.php:49
20	0.0743	9149360	dokuwiki\plugin\struct\meta\SearchConfigParameters->resolveColumn( )	.../SearchConfigParameters.php:120
21	0.0743	9149360	dokuwiki\plugin\struct\meta\Search->findColumn( )	.../SearchConfigParameters.php:66
```

**Comment by splitbrain** [comment]
I think the best way to fix that is to make the SearchConfig silently ignore unknown filters instead of throwing an error.

**Comment by micgro42** [comment]
Should be fixed by #108.
